__tests__/app.test.js

```diff
@@ -81,7 +81,49 @@ describe('Endpoint Tests', () => {
                     expect(body).toBeSorted({key: "created_at", descending: true})
                 })
             });
-
+            test('GET 200   | Returns an object ordered ascending if queried', () => {
+                return request(app).get('/api/articles?order_by=asc').expect(200).then(({body}) => {
+                    body = body.articles
+                    expect(body).toBeSorted({descending: false})
+                })
+            });
+            test('GET 200   | Returns an object ordered by authors & descending when only passed authors', () => {
+                return request(app).get('/api/articles?sort_by=author').expect(200).then(({body}) => {
+                    body = body.articles
+                    expect(body).toBeSortedBy("author", {descending: true})
+                })
+            });
+            test('GET 200   | Returns objects with only coding topics when passed the query', () => {
+                return request(app).get('/api/articles?topic=coding').expect(200).then(({body}) => {
+                    body = body.articles
+                    expect(body).toBeSortedBy("created_at", {descending: true})
+                    body.forEach(article => {
+                        expect(article.topic).toBe("coding")
+                    })
+                })
+            });
+            test('GET 200   | Returns empty array when valid topic is passed but no results are available', () => {
+                return request(app).get('/api/articles?topic=swimming').expect(200).then(({body}) => {
+                    body = body.articles
+                    expect(body.length).toBe(0)
+                })
+            });
```

Comment:
This one should be a 404, swimming isn't a topic that exists even if it is valid. 

However you do want a 200 test for a topic that exists but has no articles yet. I think paper has no articles...

So either way, amend this test and then add another!